packages/experiments/src/components/Coachmark/Coachmark.tsx

```diff
@@ -0,0 +1,226 @@
+// Utilities
+import * as React from 'react';
+import { BaseComponent } from 'office-ui-fabric-react/lib/Utilities';
+
+// Component Dependencies
+import { PositioningContainer } from '../PositioningContainer/PositioningContainer';
```

Comment:
Maybe you should have a top-level export for PositioningContainer if it's meant to be a reusable separate component, and import it from that top-level import.